src/app/public/modules/file-attachment/fixtures/file-attachment.component.fixture.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Component,
+  ViewChild,
+  OnInit
+} from '@angular/core';
+
+import {
+  SkyFileAttachmentComponent
+} from '../file-attachment.component';
+
+import { FormGroup, FormControl, Validators, FormBuilder } from '@angular/forms';
+
+@Component({
+  selector: 'file-attachment-test',
+  templateUrl: './file-attachment.component.fixture.html'
+})
+export class FileAttachmentTestComponent implements OnInit {
+  @ViewChild(SkyFileAttachmentComponent)
+  public fileAttachmentComponent: SkyFileAttachmentComponent;
+
+  public showLabel: boolean = true;
+
+  public fileForm: FormGroup;
+
+  public attachment: FormControl;
```

Comment:
Code Style > public properties should be in alphabetical order, with `ViewChild` at the bottom.